src/evaluate.py

```diff
@@ -61,6 +63,13 @@ def main():
             metrics_list.append(lang_metrics)
     
     metrics_df = pd.DataFrame(metrics_list)
+
+    results_dir = f"./results/{config.task.task_name}/{config.eval.num_fewshot}/" if config.eval.num_fewshot > 0 else f"./results/{config.task.task_name}/0"
```

Comment:
I think `f"./results/{config.task.task_name}/{config.eval.num_fewshot}/"` would do, do we really need the if?

Reply:
Also just an idea, why not log this directly to Wandb as a table? I think we can do that

Reply:
This is a temporary fix. I'm fine with logging directly to W&B if it doesn't disrupt the leaderboard pipeline flow. Ideally, the leaderboard should report the average of the 5 prompts.